dev_tools/triage-party/kubernetes/02_deployment/configmap.yaml

```diff
@@ -77,6 +84,28 @@ data:
           - issue-needs-discoverability
 
     rules:
+      ## Cirq 1.0 Release.
+      issue-needs-time:
+        name: "Untimed issues"
+        resolution: "Add a time/* label"
+        type: issue
+        filters:
+          - label: "!time/.*"
+
+      issues-before-one:
+        name: "Issues before Cirq 1.0"
+        resolution: "Open a PR to close these issues"
+        type: issue
+        filters:
+          - label: "time/before-one"
```

Comment:
Optional nit: is it acceptable to use digits? Then we could name this "time/before-1.0" maybe which is more future-proof?